lua/ts-node-action/actions/toggle_multiline.lua

```diff
@@ -1,33 +1,24 @@
 local helpers = require("ts-node-action.helpers")
 
 local function collapse_child_nodes(padding)
```

Comment:
pass optional list of node types that _have_ children (ruby ternary) but shouldn't be collapes.